app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -10,6 +10,40 @@ def initialize(file_set, user)
         @user = user
       end
 
+      # @!group Asynchronous Operations
+
+      # Spawns asynchronous IngestJob
+      # Called from FileSetsController, AttachFilesToWorkJob, ImportURLJob, IngestLocalFileJob
+      # @param [Hyrax::UploadedFile, File, ActionDigest::HTTP::UploadedFile, Tempfile] file the file uploaded by the user
```

Comment:
Yes, the wrapper thing makes sense. I'm just calling out a potential problem with the Yard documentation.  I don't think we want to accept a Tempfile, because they aren't going to have much success with it (even though the exception wouldn't occur until the background job)